src/as/__tests__/as-pect.d.ts

```diff
@@ -1,482 +1 @@
-	 *
-	 * @type {Expectation<T>}
-	 */
-	not: Expectation<T>
-
-	/**
-	 * The actual value of the expectation.
-	 */
-	actual: T | null
-	private _not: boolean
-}
-
-/**
- * This is called to stop the debugger.  e.g. `node --inspect-brk asp`.
- */
-declare function debug(): void
-
-/**
- * This function call enables performance statistics gathering for the following test.
- *
- * @param {bool} enabled - The bool to indicate if performance statistics should be gathered.
- */
-declare function performanceEnabled(enabled: bool): void
-
-/**
- * This function call sets the maximum number of samples to complete the following test.
- *
- * @param {f64} count - The maximum number of samples required.
- */
-declare function maxSamples(count: f64): void
-
-/**
- * This function call sets the number of decimal places to round to for the following test.
- *
- * @param {i32} deicmalPlaces - The number of decimal places to round to
- */
-declare function roundDecimalPlaces(count: i32): void
-
-/**
- * This function call will set the maximum amount of time that should pass before it can stop
- * gathering samples for the following test.
- *
- * @param {f64} time - The ammount of time in milliseconds.
- */
-declare function maxTestRunTime(time: f64): void
-
-/**
- * This function call enables gathering the average/mean run time of each sample for the following
- * test.
- *
- * @param {bool} enabled - The bool to indicate if the average/mean should be gathered.
- */
-declare function reportAverage(enabled: bool): void
-
-/**
- * This function call enables gathering the median run time of each sample for the following test.
- *
- * @param {bool} enabled - The bool to indicate if the median should be gathered.
- */
-declare function reportMedian(value: bool): void
-
-/**
- * This function call enables gathering the standard deviation of the run times of the samples
- * collected for the following test.
- *
- * @param {bool} enabled - The bool to indicate if the standard deviation should be gathered.
- */
-declare function reportStdDev(value: bool): void
-
-/**
- * This function call enables gathering the largest run time of the samples collected for the
- * following test.
- *
- * @param {bool} enabled - The bool to indicate if the max should be gathered.
- */
-declare function reportMax(value: bool): void
-
-/**
- * This function call enables gathering the smallest run time of the samples collected for the
- * following test.
- *
- * @param {bool} enabled - The bool to indicate if the min should be gathered.
- */
-declare function reportMin(value: bool): void
-
-/**
- * This function call enables gathering the varaince of the samples collected for the following test.
- *
- * @param {bool} enabled - The bool to indicate if the variance should be calculated.
- */
-declare function reportVariance(value: bool): void
+/// <reference path="../../../node_modules/@as-pect/assembly/types/as-pect.d.ts" />
```

Comment:
I haven't used `reference` comments before. Is this better than adding the file in tsconfig `files`?

Reply:
All I did was run `npx asp --types` to clean up the types declaration. There's no reason to expose those types now that `as-pect` is more mature.

This was the easiest solution I could come up with to ensure good ide typings.

As for placing it in the `files` for tsconfig, if you prefer that, I can make a commit to help with that.

Reply:
All good. Was just curious. it works. :)